src/Files/ViewModels/NavToolbarViewModel.cs

```diff
@@ -974,6 +977,32 @@ public async Task CheckPathInput(string currentInput, string currentSelectedPath
             }
         }
 
+        private static async Task LaunchApplicationFromPath(string currentInput, string workingDir)
```

Comment:
Should this go in a seperate helper?

Reply:
@StefanLobbenmeier

Reply:
Not sure - since the other methods also don’t use helpers it does not make much sense to do it for just this once. I guess in the long term all possible handlers of input should implement a common interface and handle it, similar to how it already done in the backend.

But if you want it in a helper now, feel free to edit. I don’t know the Codebase that well.

Reply:
We're trying to make sure new code is better organized but I'll leave this to @d2dyno1 to decide if it's a blocker.

Reply:
I haven't touched this part yet so it's fine